pkg/cmd/server/plugin/plugin.go

```diff
@@ -46,14 +46,15 @@ func NewCommand(f client.Factory) *cobra.Command {
 				RegisterVolumeSnapshotter("velero.io/gcp", newGcpVolumeSnapshotter).
 				RegisterBackupItemAction("velero.io/pv", newPVBackupItemAction).
 				RegisterBackupItemAction("velero.io/pod", newPodBackupItemAction).
-				RegisterBackupItemAction("velero.io/serviceaccount", newServiceAccountBackupItemAction(f)).
+				RegisterBackupItemAction("velero.io/service-account", newServiceAccountBackupItemAction(f)).
```

Comment:
will these renames be a breaking change for users already using these plugins?

Reply:
No, they shouldn't be - the user doesn't currently interact with these names in any way.